### ISSUE TITLE
Locales not corret when editing labels

### ISSUE THREAD

**Issue by HTPC-Schrauber** [atrocore/atropim#528]
When editing the Labels in administration, there are always only three locales to set a label for:
en_US, de_De and ru_RU

This does not change, when configuring other locales for the UI.
In our fresh setup AtroPIM (clean, no sample data), we changed the available UI locales to en_GB and de_DE.
We are now not able to set labels for en_GB, because this is not displayed in labels. Instead there is en_US, which is useless in our case. Also ru_RU stays in label, but it is not used. 
See screenshots below.

![image](https://github.com/atrocore/atropim/assets/3968136/e373a978-ff9e-43fd-b459-ad9a4d462201)

![image](https://github.com/atrocore/atropim/assets/3968136/5ae29df4-9a5c-49ce-b368-5c88bc7c2ca6)

**Comment by HTPC-Schrauber** [comment]
OK, I found it by myself. It can be configured in --> Adminitration --> Layouts
But for a fresh install it is somewhat confusing. This should be corrected in the default layout.

**Comment by rratsun** [comment]
Yes, you are right. We will disscuss it internally and made this interface more user friendly.
